Gruntfile.js

```diff
@@ -209,23 +209,37 @@ module.exports = function(grunt) {
       svgOutput: [
         "config/*SVG.js",
         "config/*SVG-full.js",
-        "unpacked/config/*HTMLorMML.js",
-        "unpacked/config/*HTMLorMML-full.js",
+        "unpacked/config/*SVG.js",
+        "unpacked/config/*SVG-full.js",
         "jax/output/SVG",
         "unpacked/jax/output/SVG"
       ],
       commonHtmlOutput: [
-        "configs",
-        "unpacked/config/",
+        "configs/*CHTML.js",
+        "configs/*CHTML-full.js",
+        "unpacked/config/*CHTML.js",
+        "unpacked/configs/*CHTML-full.js",
         "jax/output/CommonHTML",
         "unpacked/jax/output/CommonHTML",
         "extensions/CHTML-preview.js",
         "unpacked/extensions/CHTML-preview.js"
       ],
+      previewHtmlOutput: [
+        "jax/output/PreviewHTML",
+        "unpacked/jax/output/PreviewHTML",
+        "extensions/fast-preview.js",
+        "unpacked/extensions/fast-preview.js"
+        "extensions/CHTML-preview.js",
+        "unpacked/extensions/CHTML-preview.js"
```

Comment:
Are these lists of files to be *kept* or ones to be *removed*?  Note that `CHTML-preview.js` is a legacy file that is never needed for new installations (it is only there for people who included it before CommonHTML was finished and we renamed the old CommonHTML to PreviewHTML).  The old `CHTML-preview.js` is now `fast-preview.js`.  So you don't needed to include `CHTML-preview.js` here or above, I think.

Also, is the previewHtmlOutput block used for including fast preview in the build, or for using PreviewHTML as the output renderer?  If the latter, you don't need `fast-preview.js` here.